packages/client/App.tsx

```diff
@@ -53,12 +65,13 @@ const client = new ApolloClient({
 
 export type RootStackParamList = {
   MapView : Parameters<typeof MapView>[0]
-  AccountView : Parameters<typeof AccountView>[0]
+  AccountView : undefined
```

Comment:
This type is uninformative. Why did it change?

Reply:
My mistake, I will change it back.